server/discussion/queries.ts

```diff
@@ -124,7 +125,14 @@ export const createDiscussion = async (options: CreateDiscussionOpts) => {
 	});
 
 	if (commenterId) {
-		await Discussion.update({ commenterId });
+		await Discussion.update(
+			{ commenterId },
+			{
+				where: {
+					id: newDiscussion.id,
+				},
+			},
+		);
```

Comment:
`Model.update` needs a `where`. But I don't think this ever gets called, running `jest` with `coverage` shows that at least in tests this never gets called.